spec/facter/facts/windows/processors/threads_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+describe Facts::Windows::Processors::Threads do
+  describe '#call_the_resolver' do
+    it 'returns a fact' do
+      expected_fact = instance_double(Facter::ResolvedFact, name: 'processors.threads', value: 'value')
+      allow(Facter::Resolvers::Processors).to receive(:resolve).with(:threads_per_core).and_return('value')
```

Comment:
Same as my other comment about keeping mock values close to reality. I know we test the fact/resolver classes in isolation but it would be good to ensure the mocks are matching.